src/forwarders.interfaces.ts

```diff
@@ -0,0 +1,45 @@
+import { SDKEvent } from './sdkRuntimeModels';
+import { Dictionary } from './utils';
+import { IKitConfigs } from './configAPIClient';
+
+// The state of the kit when accessed via window.KitName via CDN
+// or imported as an NPM package
+export interface UnregisteredKit {
+    register(config): void;
+}
+
+// The state of the kit after being added to forwarderConstructors in the CDN 
+// or after registered to SDKConfig.kits via NPM
+export interface RegisteredKit {
+    constructor(): void;
+    name: string;
+    getId(): number;
+}
+
+// The state of the kit aftering being configured. This is what the kit looks like when acted on.
+export interface ConfiguredKit
+    extends Omit<IKitConfigs, 'isDebugString' | 'hasDebugString'> {
+    common: Dictionary;
+    init(settings, service, testMode, trackerId, userAttributes, userIdentities, appVersion, appName, customFlags, clientId): string;
+    onIdentifyComplete(user, filteredIdentityRequest): string | KitMappedMethodFailure;
+    onLoginComplete(user, filteredIdentityRequest): string | KitMappedMethodFailure;
+    onLogoutComplete(user, filteredIdentityRequest): string | KitMappedMethodFailure;
+    onModifyComplete(user, filteredIdentityRequest): string | KitMappedMethodFailure;
+    onUserIdentified(user): string | KitMappedMethodFailure;
+    process(event: SDKEvent): string;
+    setOptOut(isOptingOut: boolean): string | KitMappedMethodFailure;
+    removeUserAttribute(key:string): string;
+    setUserAttribute(key: string, value:string): void;
```

Comment:
Based on usage, this returns a result.

Reply:
Weird. I thought I updated this when I changed removeUserAttribute.  Good catch.